docs/source/overview.rst

```diff
@@ -44,7 +44,7 @@ Workflow Diagrams
 
 
 .. toggle-header::
-    :header: **2. Using Multiple Source Repos:**
+    :header: **2. Workflow Using Multiple Source Repos:**
 
       Workflow using the Mutliple Source Repos w/ Merge Cloudformation Template
```

Comment:
Diagram does not load here :( When the link is clicked it also returns 404.

Reply:
Not exactly sure why this change would cause it not to load, am trying to troubleshoot this now.

Reply:
The image now loads, but clicking on them for full size now returns 404 because they link to: https://mozdef-event-framework.readthedocs.io/en/rtd_buildout/overview.rst

Reply:
I had added a target: but it's removed now.